src/error.rs

```diff
@@ -77,6 +106,14 @@ impl fmt::Display for Error {
             Kind::Json(ref e) => fmt::Display::fmt(e, f),
             Kind::TooManyRedirects => f.write_str("Too many redirects"),
             Kind::RedirectLoop => f.write_str("Infinite redirect loop"),
+            Kind::ClientError(ref code) => {
+                f.write_str("Client Error: ")?;
+                fmt::Display::fmt(code, f)
+            }
+            Kind::ServerError(ref code) => {
+                f.write_str("Server Error: ")?;
```

Comment:
I think so. I suspect that this will be shown to end-users who might not be familiar with the different HTTP status codes. (Note that I was just copying the [python version](https://github.com/kennethreitz/requests/blob/84099eea9f99168b73ff48270dcab24e5a1ee959/requests/models.py#L921-L925) here